### PR TITLE
Update istioctl pc workloads instruction

### PR DESCRIPTION
**Pull request by linsun**
**Please provide a description of this PR:**

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**
Talked to @GregHanson, the port number should be 15021.  

```
k exec -it $ZTUNNEL -n istio-system -- curl localhost:15021/config_dump > ztunnel-config.json
Defaulted container "istio-proxy" out of: istio-proxy, debugger-xbvl8 (ephem), istio-init (init)
```
- [x ] Ambient
- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: istioctl/cmd/proxyconfig.go [hunk 1]
```diff
@@ -576,7 +576,7 @@ func workloadConfigCmd() *cobra.Command {
   istioctl proxy-config workload <pod-name[.namespace]> --type HTTP --address 0.0.0.0 -o json
 
   # Retrieve workload summary without using Kubernetes API
```
Comment (howardjohn) [inline istioctl/cmd/proxyconfig.go:578]: this isn't really "not using Kubernetes API"

**Comment by linsun** [comment]
> Its going to change to 15000 in [istio/ztunnel#243](https://github.com/istio/ztunnel/pull/243) fwiw

Thanks John, I changed back to 15000.

**Review by GregHanson — APPROVED** [review]

File: istioctl/cmd/proxyconfig.go (hunk 1 above)
Comment (linsun) [reply istioctl/cmd/proxyconfig.go]: updated!

**Comment by linsun** [comment]
Ok to approve this @howardjohn or @dhawton ?

**Comment by linsun** [comment]
Tried this cmd yesterday, still good to have the instruction updated

**Review by dhawton — APPROVED** [review]

**Pull request merged**